sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -2921,6 +2922,11 @@ pi_result piPluginInit(pi_plugin *PluginInit) {
   // PI interface supports higher version or the same version.
   strncpy(PluginInit->PluginVersion, SupportedVersion, 4);
 
+  // Set whole function table to zero to make it easier to detect if
+  // functions are not set up below.
+  std::memset(&(PluginInit->PiFunctionTable), 0,
+              sizeof(PluginInit->PiFunctionTable));
+
 // Forward calls to OpenCL RT.
```

Comment:
OpenCL RT, really? )

Reply:
@bjoernknafla, would you like to address this comment before the merge or after?

Reply:
Never mind, it will be addressed by https://github.com/intel/llvm/pull/1241.

Reply:
Good catch and thank you for tracking it being fixed by @fwyzard !